charts/fybrik/values.yaml

```diff
@@ -266,7 +272,18 @@ opaServer:
   #   log: |-
   #     package system.log
   #     mask["/input"]
-  bootstrapPolicies: {}
+
+  # Bootstrap policies to load upon startup
+  bootstrapPolicies:
+    notebookSamplePolicy: |-
+      package dataapi.authz
```

Comment:
Why is notebookSamplePolicy defined in values.yaml and not in notebook-tests.values.yaml?

Reply:
My mistake.  I moved it there temporarily when I was checking something.  Will remove.